src/WebJobs.Script.WebHost/Security/Key.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public class Key : IEquatable<Key>
+    {
+        public Key()
+        {
+        }
+
+        public Key(string name, string value)
+        {
+            Name = name;
+            Value = value;
+        }
+
+        [JsonProperty(PropertyName = "name")]
+        public string Name { get; set; }
+
+        [JsonProperty(PropertyName = "value")]
+        public string Value { get; set; }
+
+        [JsonProperty(PropertyName = "encrypted")]
+        public bool IsEncrypted { get; set; }
```

Comment:
@brettsam had the same question. Here's what I've replied with:

> I believe the intent is clearer with the flag in place. Looking at a file with an empty key would require people to understand how the code is dealing with that case, make assumptions that may turn out to be incorrect, or worse, have to read documentation (you know how people feel about that :))
Also, I was thinking that if we were to interpret an empty/null/missing key id in the document though, it would make sense that having the encryption flag set to true and the key id in one of those states would just mean "use the default key"

I feel having the flag in place provides clarity, and gives us the option to handle scenarios where the encryption key is not set differently, but since both of you have raised this issue, I'm open to changing it if you guys don't agree or feel strongly about not having the flag.